website/source/docs/internals/scheduling.html.md

```diff
@@ -55,7 +55,7 @@ Nomad servers run scheduling workers, defaulting to one per CPU core, which are
 process evaluations. The workers dequeue evaluations from the broker, and then invoke
 the appropriate scheduler as specified by the job. Nomad ships with a `service` scheduler
 that optimizes for long-lived services, a `batch` scheduler that is used for fast placement
-of batch jobs, and a `core` scheduler which is used for internal maintenance. Nomad can
+of batch jobs, a `system` scheduler that is used to run jobs globally (in every node), and a `core` scheduler which is used for internal maintenance. Nomad can
```

Comment:
Lets get rid of the parenthesis and just go with `run jobs on every node, and a core...". Can you also format the long line